input/data/2006/2006A%26A...448L..43A/tev-000037.yaml

```diff
@@ -1,4 +1,24 @@
 source_id: 37
 reference_id: 2006A&A...448L..43A
 
-# TODO: add data
+data:
+  excess: 2152
+  significance: 16.7
+
+pos:
+  ra: {val: 8h35m1s, err: 2m}
+  dec: {val: -45d34m40s, err: 2m}
+
+morph: 
+  type: gauss
+  sigma: {val: 0.48d, err: 0.03d}
+  sigma2: {val: 0.36d, err: 0.03d}
+  pa: {val: 41d, err: 7d}
```

Comment:
Please leave a comment above position angle:
```
TODO: check position angle (RADEC or GALACTIC? orientation?)
```
Deciding and documenting this will take a bit longer, and shouldn't hold up this pull request.